cmd/generate-plugin-overview/generate-plugin-overview.go

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019 The Kubernetes Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// plugin-overview reads the manifests in a directory and creates a markdown overview page
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/golang/glog"
```

Comment:
maybe don't import glog (and just use log.Printf+os.Exit) if it helps you avoid glog hijacking flags

Reply:
I tried, but it didn't really work. It's fine either way. As you said, the tool is just a little helper.